[2024-10-15T19:36:09.557+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-15T19:36:10.066+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_world_dag.hello_world_task2 scheduled__2024-09-20T00:00:00+00:00 [queued]>
[2024-10-15T19:36:10.129+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_world_dag.hello_world_task2 scheduled__2024-09-20T00:00:00+00:00 [queued]>
[2024-10-15T19:36:10.146+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-15T19:36:10.233+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): hello_world_task2> on 2024-09-20 00:00:00+00:00
[2024-10-15T19:36:10.289+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3341) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-15T19:36:10.311+0000] {standard_task_runner.py:63} INFO - Started process 3397 to run task
[2024-10-15T19:36:10.312+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'hello_world_dag', 'hello_world_task2', 'scheduled__2024-09-20T00:00:00+00:00', '--job-id', '729', '--raw', '--subdir', 'DAGS_FOLDER/example_hello_world.py', '--cfg-path', '/tmp/tmp5_0gxcvr']
[2024-10-15T19:36:10.348+0000] {standard_task_runner.py:91} INFO - Job 729: Subtask hello_world_task2
[2024-10-15T19:36:10.658+0000] {task_command.py:426} INFO - Running <TaskInstance: hello_world_dag.hello_world_task2 scheduled__2024-09-20T00:00:00+00:00 [running]> on host f9b6cc3118f7
[2024-10-15T19:36:11.409+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hello_world_dag' AIRFLOW_CTX_TASK_ID='hello_world_task2' AIRFLOW_CTX_EXECUTION_DATE='2024-09-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-20T00:00:00+00:00'
[2024-10-15T19:36:11.414+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-15T19:36:11.418+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-15T19:36:11.421+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'Hello World! Now I am triggered after the first task!'"]
[2024-10-15T19:36:11.472+0000] {subprocess.py:86} INFO - Output:
[2024-10-15T19:36:11.481+0000] {subprocess.py:93} INFO - Hello World! Now I am triggered after the first task!
[2024-10-15T19:36:11.482+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-15T19:36:11.484+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-15T19:36:11.576+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=hello_world_dag, task_id=hello_world_task2, run_id=scheduled__2024-09-20T00:00:00+00:00, execution_date=20240920T000000, start_date=20241015T193610, end_date=20241015T193611
[2024-10-15T19:36:11.754+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-15T19:36:11.846+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-15T19:36:11.869+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
